app/jobs/inherit_permissions_job.rb

```diff
@@ -18,7 +19,13 @@ def perform(work)
 
       # apply the new and deleted attributes
       file.permissions_attributes = attribute_map
-      file.save!
+      persister.save(resource: file)
```

Comment:
Is this just an result of systematically removing `.save!` in favor of the Valkyrie way?  I'm curious as to why not just go the rest of the way and use a `change_set` and a `change_set_persister` over this?  My understanding of the pattern was to keep the persister.save calls into change_set_persisters and prefer saving through them, eg `change_set_persister.save(change_set: change_set)`, whenever possible.  

In Figgy, I'm seeing the pattern for jobs:

https://github.com/pulibrary/figgy/blob/0d6aa5f62237537085e61922eb5e39d5bacd51ca/app/jobs/browse_everything_ingest_job.rb#L5-L15

Reply:
Thanks I'll look into that.  My intention right now is to make broken code not broken for the tests and then later come back to doing it the right way.